api-auto-registration/installation.hbs.md

```diff
@@ -9,6 +9,11 @@ This topic describes how to install API Auto Registration from the Tanzu Applica
 Before installing API Auto Registration, complete all prerequisites to install Tanzu Application Platform.
 See [Tanzu Application Platform Prerequisites](../prerequisites.md).
 
+## <a id='prereqs'></a>Notes on TLS
+Starting in TAP 1.4 TLS is turned on by default for several TAP components. This caused an issue with API Auto Registration, more details and a work around can be found [here](../api-auto-registration/troubleshooting.hbs.md). 
+
+Starting in TAP 1.4.1 a change was introduced that allows API Auto Registration to automatically trust the CA for the shared ingress_issuer. One side effect of this change is that a `Certificate` is generated using this issuer. If for some reason you do not want a `Certificate` to be generated automatically you can set the `auto_generate_cert` flag to false in the values file. You will then need to follow the instructions in the troubleshooting guide linked above to manually to set the `ca_cert_data`.
```

Comment:
@jrshaffe would it be good to show an example of where this might go?

Reply:
I added  the `auto_generate_cert` key to the example in step 3 below and then updated wording to tell them there is an example below